client/src/views/Profile.vue

```diff
@@ -34,17 +36,26 @@
               </v-card>
             </v-flex>
         </v-layout>
-    </v-container>
+    <loading-dialog ref="loadingDialog"/>
+    <response-dialog ref="responseDialog"/>
+    </v-container>    
 </template>
 
 <script>
 import axios from "axios";
+import LoadingDialog from "@/components/LoadingDialog.vue";
+import ResponseDialog from "@/components/ResponseDialog.vue";
 
 export default {
     props: ["token"],
-    data() {
+    components: {
+      LoadingDialog,
+      ResponseDialog
+    },
+    data: () => {
         return {
-            inputEmail: '',
+            inputEmail: 'mijn@email.nl',
+            userName: 'gebruikersnaam',
```

Comment:
Was het niet de bedoeling dat je deze twee dingen uit de database haalt i.p.v hardcoded erin zetten?

Reply:
is een placeholder, hij haalt het uit de db als je verder kijkt